src/lib/components/composites/navigation-bar/UserMenu.svelte

```diff
@@ -52,11 +50,8 @@
 </script>
 
 <DropdownMenu.Root>
-    <DropdownMenu.Trigger>
-        <Avatar.Root>
-            <Avatar.Fallback>{userInitials}</Avatar.Fallback>
-        </Avatar.Root>
-    </DropdownMenu.Trigger>
+    <UserAvatar {user} />
+    <DropdownMenu.Trigger></DropdownMenu.Trigger>
```

Comment:
The avatar has to be in the trigger. Otherwise, it's not clickable.
```suggestion
    <DropdownMenu.Trigger>
        <UserAvatar {user} />
    </DropdownMenu.Trigger>
```

Reply:
Oh my fault, thank you for the hint.